.gitignore

```diff
@@ -52,6 +52,9 @@ api/rest/swagger
 /vendor/
 composer.phar
 
+# Admin
+admin/install.done
```

Comment:
The `install.done` changes seem to be unrelated.  @dregad can comment on these and we should decide if they should be part of this PR or a separate one.

Reply:
Yes, it's unrelated, but me was a bit pissed of with the warning :) Maybe it would be nicer to use an install.RELASE for such purpose - making upgrades easier. Anyway, I'll wait for @dregad 's answer.

Reply:
@V1pr I can hear your point of view with the warning.

The problem with the proposed approach is that not everyone uses Apache for their webserver; someone running Mantis on IIS would no longer see the warning about the admin directory, even though it is still present and accessible.

I would prefer to treat this as a separate work item.

I'm not sure in what context the warning pissed you off, but FYI you can always disable it with `$g_admin_checks = OFF` (that's what I usually do on my dev box; obviously not recommended on a production server).

Reply:
Ok, I get your point. I'll revert these change.